### PR TITLE
Add covid keep safe message

### PR DESCRIPTION
**Pull request by GeoWill**
This seems like it might be too many words, interested to hear other's thoughts. As it's below the scroll bar on most screens I don't really mind including it though. I'd be nervous of putting it on the results page as it wouldn't get read at the bottom, and any higher and it would push other stuff way down.

Overview shot of home page: 
Screenshot from 2021-04-29 15-36-31

This is what the card looks like:
Screenshot from 2021-04-29 15-35-13

### CODE REVIEW

File: polling_stations/templates/fragments/keep_safe.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% load i18n_with_welsh %}
+  <div class="card centered-card">
```
Comment (symroe) [inline polling_stations/templates/fragments/keep_safe.html:2]: Can we get rid of `centered-card` please?

File: polling_stations/templates/fragments/keep_safe.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        <li>{% trans "Please remember your face covering- you will need this to enter the polling station and will be expected to wear it throughout." %}</li>
+        <li>{% trans "Make sure to sanitise your hands before and after voting. In many places, sanitiser will be available but it may be a good idea to carry your own!" %}</li>
+        <li>{% trans "Some polling stations will supply clean pencils for voters but it may be best to bring your own with you." %}</li>
+        <li>{% trans "Polling Stations will be cleaned regularly by polling staff. You may need to wait while a booth is sanitised before you enter." %}</li>
+        <li>{% trans "Polling Station staff may be working behind safety screens- this doesn’t mean you can’t ask for assistance if you require it!" %}</li>
+      </ul>
+      <p>{% trans "With these new safety measures in place, it may take a little bit longer to vote than usual. Remember that if you are in the queue to vote before 10pm, you will still be able to vote even when polls ‘officially close’." %}</p>
```
Comment (symroe) [inline polling_stations/templates/fragments/keep_safe.html:14]: Why the scare quotes?

Comment (GeoWill) [reply polling_stations/templates/fragments/keep_safe.html:14]: Good question! Not needed.

**Comment by GeoWill** [comment]
Fixups: Screenshot from 2021-04-29 16-16-40

**Pull request merged**